src/app/memories/page.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export default function Page() {
+
+    return (
+        <div className="container mx-auto max-w-3xl py-16">
+            <h1 className="text-4xl font-bold mb-8">
+                Share Your Memories
+            </h1>
+
+            <div className="mb-8 space-y-4">
+                <p>
+                    We would love to hear from you! Share your memories and photos with us to help build the story of our community.
+                </p>
+                <p>
+                    Send your contributions to: <a href="mailto:kamil.dzieniszewski@gmail.com" className="font-semibold">contact@example.com</a>
```

Comment:
Should the displayed and mailto emails match?

Reply:
true, wrong email